website/docs/installation/upgrade.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Upgrade authentik
```

Comment:
We should state somewhere that upgrading is only supported from minor to minor (for instance 2024.2 to 2024.4 is supported, but 2024.2 to 2024.6 is not).